src/components/productlistloadmore.jsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+import React from 'react';
+import { withRouter } from 'react-router';
+import PropTypes from 'prop-types';
+import intl from 'react-intl-universal';
+import { navigationLookup } from '../utils/CortexLookup';
+import { login } from '../utils/AuthService';
+
+import './productlistloadmore.less';
+
+class ProductListLoadMore extends React.Component {
+  static propTypes = {
+    dataProps: PropTypes.objectOf(PropTypes.any).isRequired,
+    handleDataChange: PropTypes.func.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+    const nextLink = props.dataProps.links.find(link => link.rel === 'next');
+    this.state = {
+      canLoadMore: Boolean(nextLink),
+      isLoading: false,
+    };
+
+    this.loadMore = this.loadMore.bind(this);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const nextLink = nextProps.dataProps.links.find(link => link.rel === 'next');
+    this.setState({ canLoadMore: Boolean(nextLink) });
+  }
+
+  loadMore() {
+    const { dataProps, handleDataChange } = this.props;
+    this.setState({ isLoading: true });
+    login().then(() => {
+      const nextRel = dataProps.links.find(link => link.rel === 'next');
+      if (nextRel) {
+        navigationLookup(nextRel.uri)
```

Comment:
Upon further investigation, noticed the differences. Use `searchLookup` instead of `navigationLookup` in cases where doing a search to fix the above problems.